code/obj/vehicle.dm

```diff
@@ -10,8 +10,33 @@ Contains:
 -Forklift
 */
 
+//////////////////////////////// Vehicle Defines ///////////////////////////////////////
```

Comment:
```suggestion
//------------------ Vehicle Defines --------------------///
```

Reply:
I just copied the header formatting used in other parts of the file, but yeah, I can change jt